src/lib/flexbox/api/base.ts

```diff
@@ -92,6 +91,15 @@ export abstract class BaseFxDirective implements OnDestroy, OnChanges {
   // Lifecycle Methods
   // *********************************************
 
+  /**
+   * Use post-component-initialization event to perform extra
+   * querying such as computed Display style
+   */
+  ngOnInit() {
+    this._display = this._getDisplayStyle();
+    this._hasInitialized = true;
```

Comment:
Not sure what you mean. Many of the directives using `ngOnInit()`, the hasInitialized() is a custom method used in the Flex directives.